packages/kbn-router-to-openapispec/src/oas_converter/kbn_config_schema/post_process_mutations/index.ts

```diff
@@ -39,6 +39,8 @@ const walkSchema = (ctx: IContext, schema: OpenAPIV3.SchemaObject): void => {
     mutations.processMap(ctx, schema);
   } else if (schema.type === 'string') {
     mutations.processString(schema);
+  } else if ((schema.type as string) === 'stream') {
```

Comment:
NIT: I think I asked it already in the previous PR, but I forgot the answer 😅, so I'll ask again: why is the `as string` cast necessary on this specific elif statement?